test/PublicApiTests/BaseLine/Microsoft.OData.PublicApi.net45.bsl

```diff
@@ -2990,7 +2982,6 @@ public sealed class Microsoft.OData.Edm.Csdl.SerializationExtensionMethods {
 }
 
 public class Microsoft.OData.Edm.Csdl.CsdlLocation : Microsoft.OData.Edm.EdmLocation {
-    string JsonPath  { public get; }

```

Comment:
Would rather not have something JSON-specific in format-independent EdmLocation.  Perhaps we could call this just Path?